test/model/create.test.ts

```diff
@@ -148,3 +148,24 @@ test('uses value getters when creating an entity with nested arrays', () => {
   expect(exactUser.info).toHaveProperty('tags', [1, 2])
   expect(exactUser.info).toHaveProperty('documents', [])
 })
+
+test('can create models with null values', () => {
+  const db = factory({
+    user: {
+      id: primaryKey(datatype.uuid),
+      name: nullable(name.findName),
+      address: {
+        street: String,
+        number: nullable<number>(() => null),
+      }
+    },
+  })
+
+  const user = db.user.create({
+    id: 'abc-123',
+    name: null,
```

Comment:
Do I understand correctly that here we're checking that the nullable property with an explicit initial value of `null` preserves that value? 

If so, we should add another case for the root-level nullable property _without_ the initial value in `.create()` to make sure it acquires the `null` value implicitly.

Reply:
Yeah that is correct, it's checking that explicit initial values of `null` are preserved. Makes sense to me, I'll add the extra case 😁